src/Pages/ReviewWrite/ReviewWrite.scss

```
@@ -0,0 +1,161 @@
+.ReviewWritePage {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 101px auto 62px;
+
+  .ReviewWrite {
+    display: flex;
+    flex-direction: column;
+    width: 678px;
+
+    .ReviewWriteSto {
+      height: 40.93px;
+      margin-bottom: 20px;
+      .ReviewWriteStoName {
```

```suggestion
      margin-bottom: 20px;
      
      .ReviewWriteStoName {
```
가독성을 위해 한칸 띄어서 작성해주세요~